spec/unit/router/routes_spec.rb

```diff
@@ -162,4 +162,46 @@ def deep_check(values)
 			it { is_expected.to be_nil }
 		end
 	end
+
+	describe '#to_s' do
+		subject { routes.to_s }
+
+		before do
+			routes['foo']['bar'][:GET] = 42
+			routes['foo']['bar']['bar'] = described_class.new
+			routes['foo']['bar']['bar'][:POST] = 36
+			routes['foo']['bar']['baz'][:DELETE] = 84
```

Comment:
If one 'baz' will be above 'bar' here, it will increase verbosity of sorting

Reply:
Done, thank you.